cli/test/e2e/package_authoring_e2e_test.go

```diff
@@ -15,26 +15,105 @@ import (
 )
 
 const (
-	workingDir = "kcrl-test"
+	workingDir = "kctrl-test"
 )
 
-func TestPackageInitAndRelease(t *testing.T) {
-	env := BuildEnv(t)
-	logger := Logger{}
-	kappCtrl := Kctrl{t, env.Namespace, env.KctrlBinaryPath, logger}
+type E2EAuthoringTestCase struct {
```

Comment:
Instead of having a separate list of outputs, how about we add a key for `ExpectedOutput` in `E2ETestAuthoringCase`, and store expected output in the test case itself.

Reply:
+1

Reply:
Done